apps/ledger-live-desktop/src/renderer/modals/SignTransaction/Body.tsx

```diff
@@ -134,7 +136,10 @@ export default function Body({ onChangeStepId, onClose, setError, stepId, params
   } = useBridgeTransaction(() => {
     const parentAccount = params && params.parentAccount;
     const account = params && params.account;
-    const bridge = getAccountBridge(account, parentAccount);
+    const bridge = params.isACRE
+      ? // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        (ACREBridge.accountBridge as unknown as AccountBridge<any>)
+      : getAccountBridge(account, parentAccount);
```

Comment:
I don't think we actually need this change as it's not doing anything special with the `hw-app-acre`